import/webview/WebView.qml

```diff
@@ -97,7 +98,9 @@ RawWebView {
     }
 
     onRecvAsyncMessage: {
-        if (pickerOpener.message(message, data) || popupOpener.message(message, data)) {
+        if (pickerOpener.message(message, data)
+                || popupOpener.message(message, data)
+                || PermissionManager.message(message, data)) {
```

Comment:
I guess we need to opt in this handling here due to in app usage and we don't necessarily want to handle permissions. See other comments as well.

Reply:
Further this looks none view specific on the https://git.sailfishos.org/mer-core/embedlite-components/merge_requests/63/diffs#4eb83ed266b2098cf49b438ad1f855e80bdafa27_0_36

WebEngine's notifyObservers and addObserver(s) should be enough here.

Reply:
Looked at notifyObservers and did not find how to use it to catch the data signal when requesting the list of permissions, could you give more information please

Reply:
There's  "void recvObserve(const QString message, const QVariant data);" signal in the WebEngine. You can connect to that in the model itself. In addition, you should call in the model WebEngine::addObserver("mytopic"); to register listening messages.